test/presubmit-tests.sh

```diff
@@ -32,6 +32,8 @@
 # Serving / Eventing of this specific version.
 
 export DISABLE_MD_LINTING=1
+export DISABLE_MD_LINK_CHECK=1
```

Comment:
Can you add a TODO here? If we have a chance to fix that issue in liche in the future, we can reenable it.

Reply:
Sorry! forgot to add TODO, I'd filed https://github.com/raviqqe/liche/issues/33

Reply:
No worries. Thanks.